thoth/graph_metrics_exporter/configuration.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+# thoth-graph-metrics-exporter
+# Copyright(C) 2021 Francesco Murdaca
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Configuration for all graph metrics exporter tasks."""
+
+import os
+
+
+from prometheus_client import CollectorRegistry
+
+
+class Configuration:
```

Comment:
Environment variables can be directly be accessed (e.g. global scope) one this gets merged into a single file - e.g. graph env vars are unused.

Reply:
same as above, there are differents tasks accessing common logic

Reply:
https://github.com/thoth-station/graph-metrics-exporter/pull/3#discussion_r653574781